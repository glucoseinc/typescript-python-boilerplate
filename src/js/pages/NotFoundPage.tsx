import React from 'react'

const NotFoundPage: React.FunctionComponent<{}> = () => {
  return <div>Not Found</div>
}

export default NotFoundPage
