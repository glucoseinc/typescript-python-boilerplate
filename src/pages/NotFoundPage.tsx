import React from 'react'

const NotFoundPage: React.FunctionComponent<{}> = () => {
  return <main>Not Found</main>
}

export default NotFoundPage
