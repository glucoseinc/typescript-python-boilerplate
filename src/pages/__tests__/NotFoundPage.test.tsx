import {mount} from 'enzyme'
import * as React from 'react'

import NotFoundPage from '../NotFoundPage'

describe('NotFoundPage', () => {
  test('renders correctly', () => {
    const element = mount(<NotFoundPage />)

    expect(element.find('NotFoundPage')).toMatchSnapshot()
  })
})
