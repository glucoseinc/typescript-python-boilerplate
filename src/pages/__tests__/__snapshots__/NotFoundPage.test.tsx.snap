// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<NotFoundPage>
  <main>
    Not Found
  </main>
</NotFoundPage>
`;
